server/src/datastore/timesheet.repo.ts

```diff
@@ -11,6 +11,14 @@ export class TimesheetRepository extends Repository<Timesheet> {
       .getRawMany();
   }
 
+  lastEntryByUser(): any {
+    return this.createQueryBuilder('time')
+      .groupBy('time."user_id"')
+      .select('time."user_id"')
+      .addSelect('MAX(time.date)', 'date')
+      .getRawMany();
```

Comment:
If we are expecting only one value, we can use `getRawOne`: https://typeorm.io/#/select-query-builder/getting-values-using-querybuilder